packages/typespec-python/tasks.py

```diff
@@ -118,9 +118,9 @@
     "client/structure/two-operation-group": {
         "package-name": "client-structure-twooperationgroup",
     },
-    "mgmt/sphere": [
-        {"package-name": "azure-mgmt-spheredpg"},
-    ],
+    # "mgmt/sphere": [
+    #     {"package-name": "azure-mgmt-spheredpg"},
+    # ],
```

Comment:
Because typespec-azure-resource-manager 0.56.0 has breaking https://github.com/Azure/typespec-azure/blob/main/packages/typespec-azure-resource-manager/CHANGELOG.md#deprecations, we have to skip the test for real service sphere. After sphere fix it, we will update the typespec and open it.